doc/MissingFeatures.md

```diff
@@ -0,0 +1,43 @@
+## Missing Features
+
+The following is a list of features missing to match the support from [oneAPI oneMKL BLAS](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/domains/blas/blas).
+The order does not imply any priority.
+
+For questions regarding input types or operators support, please refer to this [documentation](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/domains/blas/blas)
+
+- Add row-major support to level-1 operators
+- Add row-major support to level-2 operators
+- Add row-major support to level-3 operators
+- Add complex support to level-1 operators that required it.(asum, axpy, copy, nrm2, rot, rotg, scal, swap, iamax,iamin)
```

Comment:
Thanks for your patch, @s-Nick.

```suggestion
- Add complex support to level-1 operators that required it: asum, axpy, copy, nrm2, rot, rotg, scal, swap, iamax, iamin.
```

Reply:
Thank you for your feedback @pgorlani , I addressed them in [8556c2a](https://github.com/codeplaysoftware/portBLAS/pull/534/commits/8556c2a2b404fc6222fb8f9752f397fd5e212691)